src/NinjaTools.FluentMockServer.API/Extensions/JsonExtensions.cs

```diff
@@ -0,0 +1,66 @@
+using System;
+using System.ComponentModel;
+using System.Globalization;
+using Newtonsoft.Json.Linq;
+
+namespace NinjaTools.FluentMockServer.API.Extensions
+{
+    public static class JsonExtensions
+    {
+        public static bool IsNullOrEmpty(this JToken token)
+        {
+            return (token == null) ||
+                   (token.Type == JTokenType.Array && !token.HasValues) ||
+                   (token.Type == JTokenType.Object && !token.HasValues) ||
+                   (token.Type == JTokenType.String && token.ToString() == String.Empty) ||
+                   (token.Type == JTokenType.Null);
+        }
+
+        public static bool IsValidJson(this string text)
+        {
+            text = text.Trim();
+            if ((text.StartsWith("{") && text.EndsWith("}")) || //For object
+                (text.StartsWith("[") && text.EndsWith("]"))) //For array
+            {
+                try
+                {
+                    var obj = JToken.Parse(text);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove the unused local variable 'obj'.](https://app.codacy.com/manual/git_36/NinjaTools.FluentMockServer/pullRequest?prid=4802642)